justfile

```
@@ -1,4 +1,4 @@
-ci_build_image := "jamwaffles/circleci-embedded-graphics:1.40.0-3"
+ci_build_image := "jamwaffles/circleci-embedded-graphics:1.56.1"
```

I would always add the suffix to keep the same tag format.
```suggestion
ci_build_image := "jamwaffles/circleci-embedded-graphics:1.56.1-0"
```

---

I started with `-0` and decided to remove it, but yeah let's keep it for consistency 👍